.github/workflows/check.yml

```diff
@@ -89,3 +89,19 @@ jobs:
           max-attempts: 5
           retry-delay: 2s
           retry-all: true
+
+  check-single-with-basic-auth:
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+    runs-on: ${{ matrix.os }}
+    name: Check single, with cookie
```

Comment:
The cookie option doesn’t seem to be actually set below. Can I ask you provide a test for the cookie too?

Reply:
It's just a typo. Fixed.

I've tried to find any public test endpoint which requires cookie for auth, but unfortunately unsuccessful.
So I just checked it locally.

If you have any suitable endpoint or probably know other ways to test it, please, let me know.

Thank you!

Reply:
Thanks for the context, I see what the problem is now. 

I can't think of an easy way to test them either. Using the Cookie Get echo & verifying the output comes to mind, but the step doesn't provide any `output` as of yet. I'll see if I can fix it later.

Meanwhile, I'm happy to merge the PR as it is, thank you for your contribution!